tools/replay.cpp

```diff
@@ -482,20 +479,61 @@ class Replay {
     }
 };
 
-int main(int ac, char** av)
+static cxxopts::ParseResult parse(int argc, char* argv[])
 {
-  if ( ac < 2 || ac > 3 )
-    Replay::usage_and_exit( "Incorrect number of program arguments" );
+  try
+  {
+    cxxopts::Options options(argv[0], "Replay an umpire session from a file");
+
+    options
+      .add_options()
+      (  "h, help", "Print help")
+      (  "i, infile"
+       , "Input file created by Umpire library with UMPIRE_REPLAY=On"
+       , cxxopts::value<std::string>(), "FILE"
+      )
+    ;
+
+    options.add_options("Group")
```

Comment:
Does this hide the -t option?

Reply:
Yes it does.   You tell help which group names you want to display and we do not say that it should show "Group".   I will rename "Group" to be something better.  Just a sec...

Reply:
Take a look now, this should be better.  Resolve this conversation if you agree..